istioctl/cmd/proxyconfig.go

```diff
@@ -564,16 +564,16 @@ func workloadConfigCmd() *cobra.Command {
 
 	workloadConfigCmd := &cobra.Command{
 		Use:   "workload [<type>/]<name>[.<namespace>]",
-		Short: "Retrieves workload configuration for the ztunnel in the specified pod",
-		Long:  `Retrieve information about workload configuration for the ztunnel instance in the specified pod.`,
-		Example: `  # Retrieve summary about workload configuration for a given pod from ztunnel.
-  istioctl proxy-config workload <pod-name[.namespace]>
+		Short: "Retrieves workload configuration for the specified ztunnel pod",
+		Long:  `Retrieve information about workload configuration for the ztunnel instance.`,
+		Example: `  # Retrieve summary about workload configuration for a given ztunnel.
+  istioctl proxy-config workload <ztunnel-name[.namespace]>
 
-  # Retrieve workload summary for workload with port XXXX.
-  istioctl proxy-config workload <pod-name[.namespace]> --node ambient-worker
+  # Retrieve summary of workloads on node XXXX for a given ztunnel instance.
+  istioctl proxy-config workload <ztunnel-name[.namespace]> --node ambient-worker
```

Comment:
since ztunnel has secrets, workloads, and rbac, I wonder if we should make `istioctl ztunnel-config` instead of proxy-config. Anyhow, not blocking this PR...

Reply:
agreed (cc @linsun).  along those lines, @howardjohn do you think ztunnel is at a point where we can look add secret information to the ztunnel config_dump?

Reply:
yea

Reply:
This cmd is confusing still - if a node is specified, do we still need users to specify ztunnel name?

Reply:
the way I read it:
```
 - get all the workloads a given ztunnel knows about
 - of those workloads, only display the ones located on the specified node
```

Reply:
Ok i think i understand it now. This unfortunately is quite confusing -i had thought it meant the node ztunnel runs but `--node` is just a filter of the display.